app/controllers/users_controller.rb

```diff
@@ -1,29 +1,24 @@
 class UsersController < ApplicationController
-  LOCALES = [
-              ["English", "en"],
-            ]
-
-  def settings
-    load_locales
+  def edit
```

Comment:
Actually that was a topic I wanted to bring up at stand-ups before we got distracted with staging site. I was looking to drop something into Policy model but there was a few things I thought it was worth talking about first.